apps/frontend/src/components/listingCard.tsx

```diff
@@ -92,28 +94,49 @@ const ListingCard = ({
       className="listing-card"
       onClick={handleListingClick}
     >
-      <Grid
-        item
-        sx={{
-          width: "100%",
-          maxHeight: "40vh",
-          display: "flex",
-          justifyContent: "center",
-          overflow: "hidden",
-          borderRadius: "8px",
-        }}
-      >
-        <img
-          src={listing.imageUrl !== "" ? listing.imageUrl : imageDefault}
-          alt={listing.title}
-          style={{
+      <div style={{ position: "relative", width: "100%", maxHeight: "40vh" }}>
+        {listing.charityId == "1" && (
+          <img
+            src={CharityBanner}
+            alt="charityBanner"
+            style={{
+              position: "absolute",
+              top: 0,
+              left: 0,
+              width: "100%",
+              height: "160px",
+              objectFit: "cover",
+              borderRadius: "8px",
+              zIndex: 2,
+            }}
```

Comment:
Something weird is happening with aliasing on these images, might be worth an investigation. If you'd like me to rework the image with one of my tools, just lmk!

![image](https://github.com/user-attachments/assets/fbe7617d-d590-4dfb-b908-ee12b895a760)

Reply:
Fixed